src/shared/services/challenges.js

```diff
@@ -79,7 +79,8 @@ export function normalizeMarathonMatch(challenge, username) {
     numSubmissions: challenge.userIds ? challenge.userIds.length : 0,
     platforms: '',
     prizes: [0],
-    registrationOpen: endTimestamp > Date.now() ? 'Yes' : 'No',
+    registrationOpen: endTimestamp > Date.now() &&
+      !(challenge.status && challenge.status === 'PAST') ? 'Yes' : 'No',
```

Comment:
@elfman `!(challenge.status && challenge.status === 'PAST')` is equivalent to just `challenges.status !== 'PAST'`, which is surely more clean, and thus preffered code. Will update myself, though.